### PR TITLE
Fix bad conversion of RPC.md

### PR DESCRIPTION
**Pull request by jDramaix**
Fix bad conversion of RPC.md

Bug: issue #9096 (https://code.google.com/p/google-web-toolkit/issues/detail?id=9096)

### CODE REVIEW

File: src/main/markdown/doc/latest/tutorial/RPC.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Using GWT RPC
```
Comment (branflake2267) [inline src/main/markdown/doc/latest/tutorial/RPC.md]: I'd suggest removing == and put ## or for the very first, use one # for the title.

**Comment by tbroyer** [comment]
+1 but rebase on top of `master` to remove the _Merge branch 'master' of github.com:gwtproject/gwt-site into jd_fix_RPC_page_ commit.

File: src/main/markdown/doc/latest/tutorial/RPC.md (hunk 1 above)
Comment (jDramaix) [inline src/main/markdown/doc/latest/tutorial/RPC.md]: Done

**Comment by jDramaix** [comment]
Let me done this cleanly.

**Comment by branflake2267** [comment]
Nice job

**Pull request closed without merging**